Formula/dynamodb-local.rb

```diff
@@ -67,4 +67,8 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    system "#{bin}/dynamodb-local", "-help"
```

Comment:
`-help` means Java is able to run the JAR. If the JAR had `-version` I would do that like the Elixir formula does, but it doesn't have that.

Open to suggestions if you have them